webpack.release.config.js

```diff
@@ -39,14 +39,6 @@ entryConfig.externals = {
   'u2f-api-polyfill': true,
   'underscore': true
 };
-// Add transform-runtime
-entryConfig.module.loaders = entryConfig.module.loaders || [];
```

Comment:
This was added by https://github.com/okta/okta-signin-widget/pull/353 and turns out `transform-runtime` was added twice since it's already added in `webpack.common.config.js`